src/main/java/org/hydev/mcpm/client/interaction/ProgressBar.java

```diff
@@ -56,8 +55,8 @@ public ProgressBar(ProgressBarTheme theme)
         this.frameDelay = 1 / 60d;
 
         // Check if output is a TTY. If not, change frame rate to 0.5 fps to avoid spamming a log.
-        this.istty = isatty(STDOUT_FILENO) == 0;
-        if (istty) this.frameDelay = 1 / 0.5;
+        this.istty = isatty(1) == 1;
```

Comment:
I getcha, `#define STDOUT_FILENO 1`. Any reason you removed the reference to the global constant? It makes it more clear what you're trying to do.

Reply:
Yea I was testing if changing it to STDIN_FILENO (0) would have any difference, but there isn't a constant for STDIN_FILENO so I wrote a number and forgot to change it back.